modules/regexp-parser/regexp-parser-grammar.ym

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2021 Balabit
```

Comment:
Please change "Balabit" to "One Identity" here, and in every file you have created.

Reply:
Related changes have been udpated.